### PR TITLE
[Payment] Success, cancel, findByUser endpoints

### PR DESCRIPTION
**Pull request by RedArtelerist**

### CODE REVIEW

File: src/main/java/lux/fm/bookingservice/service/impl/PaymentServiceImpl.java [hunk 1]
```diff
@@ -75,17 +80,36 @@ public PaymentResponseDto createPayment(
...
     @Override
-    public PaymentResponseDto cancel(String sessionId) {
-        return null;
+    @Transactional
+    public BookingResponseDto cancelPayment(String sessionId) {
+        Payment payment = paymentRepository.findBySessionIdAndStatus(
+                sessionId, Payment.Status.PENDING)
+                .orElseThrow(() -> new PaymentException("Payment session not found"));
+        Booking booking = payment.getBooking();
+        // @Todo: invalidate Stripe session
```
Comment (viacheslav-torbin) [inline src/main/java/lux/fm/bookingservice/service/impl/PaymentServiceImpl.java:101]: What did you meant by that?)

File: src/main/java/lux/fm/bookingservice/service/PaymentService.java [hunk 2]
```diff
@@ -14,9 +16,9 @@ PaymentResponseDto createPayment(
             UriComponentsBuilder uriComponentsBuilder
     );
 
-    BookingResponseDto success(String sessionId);
+    PaymentWithoutSessionDto successPayment(String sessionId);
```
Comment (viacheslav-torbin) [inline src/main/java/lux/fm/bookingservice/service/PaymentService.java:19]: Name is kinda confusing, but it's up to you

**Review by viacheslav-torbin — APPROVED** [review]
Left some minor comments

File: src/main/java/lux/fm/bookingservice/service/impl/PaymentServiceImpl.java (hunk 1 above)
Comment (RedArtelerist) [reply src/main/java/lux/fm/bookingservice/service/impl/PaymentServiceImpl.java:101]: Immediately expire Stripe session

**Pull request merged**